src/snapshot.rs

```diff
@@ -23,6 +23,25 @@ pub struct Snap {
     pub ledger_entries: OrdMap<LedgerKey, LedgerEntry>,
 }
 
+pub fn get_default_ledger_info() -> LedgerInfo {
+    LedgerInfo {
+        protocol_version: 19,
+        sequence_number: 0,
+        timestamp: 0,
+        network_id: vec![0u8],
```

Comment:
I think there's a sandbox network passphrase that @paulbellamy added to this repo, we should use that: 
https://github.com/stellar/soroban-cli/blob/f76bd3e68a3be0d2e87d233b887bfcd294b18987/src/network.rs#L1

Reply:
Done